_rules/aria-state-or-property-valid-value-6a7281.md

```diff
@@ -22,28 +22,19 @@ acknowledgments:
 
 ## Applicability
 
-Any [WAI-ARIA 1.1 state or property](https://www.w3.org/TR/wai-aria-1.1/#state_prop_def) that is not empty (""), and that is specified on an HTML or SVG element.
+The rule applies to any [WAI-ARIA 1.1 state or property][] that is not empty (`""`), and is specified on an HTML or SVG element.
 
 ## Expectation
 
-Each test target has a valid value according to its [WAI-ARIA 1.1 value type](https://www.w3.org/TR/wai-aria-1.1/#propcharacteristic_value).
-
-For value types `ID Reference` and `ID Reference List` for [WAI-ARIA required properties](https://www.w3.org/TR/wai-aria-1.1/#requiredState) at least one of the elements with the given ids exists in the same [document tree](https://dom.spec.whatwg.org/#document-trees) or [shadow tree](https://dom.spec.whatwg.org/#shadow-trees) as the element that specifies the target attribute.
-
-For value type `URI` the value matches the [generic URI syntax](https://www.ietf.org/rfc/rfc3986.txt).
```

Comment:
Not sure why these have suddenly been removed 🤔 

Maybe it would be clearer to have them as second and third expectation?

Reply:
@Jym77 see [issue](https://github.com/act-rules/act-rules.github.io/issues/1432), for reasoning as to why this was removed.

Reply:
If we remove that requirement, we need to clean more traces of it elsewhere in the rule… So, at least:
1. In the expectation, precise that "valid value" does not mean "existing IDref" given that the discussion in #1432 is about that.
2. Replace the Background note about "IDref must exist on required property" by something summing up #1432 and explaining why we don't require existing IDref.
3. Remove traces of it from the example descriptions. Not need to have a description about which case the IDref must or not exist since this is not part of the rule anymore.

Reply:
@Jym77, 

For:
1) I disagree, the point of moving this section to background, was not to pollute the expectation with extra details. I believe keeping expectation precise and to the point that adding edge cases is better.
2) Again, I read through the background description & seems just right. I also believe after a conclusion has been reached to #1432, perhaps this can be improved. So leaving as is.
3) I disagree. Good to have longer descriptions than leaving it open.

I will see if similar reviews are arrived from over reviewers, to take any action on them. In the meantime dismissing this.

Reply:
I agree with @Jym77 on this. ARIA 1.1 says an IDREF must exist, and for IDREFs that at least one of them must. Since that's not what this rule (should) require, we need to include an exception for it in the expectation.

Reply:
I have added a simple note.

Reply:
Notes can not contain requirements. Please put this as an exception in the expectation text.